### ISSUE TITLE
[ui5-carousel]: Accessibility aria attributes and role issues

### ISSUE THREAD

**Issue by aleksandra-beczala** [UI5/webcomponents#12506]
### Bug Description

Using UI5 Carousel component, there are accessibility issues visible in Lighthouse connected to aria-* attributes and roles for each carousel item:

<img width="850" height="398" alt="Image" src="https://github.com/user-attachments/assets/88d92afe-765b-462b-a00c-79eb1ea88ef3" />

### Affected Component

ui5-carousel

### Expected Behaviour

There should be no accessibility issues.

### Isolated Example

https://ui5.github.io/webcomponents/components/Carousel/

### Steps to Reproduce

1. Copy the first carousel example from https://ui5.github.io/webcomponents/components/Carousel/
2. Open Lighthouse in Google Chrome
3. Generate an Accessibility Lighthouse report in Snapshot mode for the Desktop device
...

### Priority

Medium

### UI5 Web Components Version

2.14

### Browser

Chrome

### Operating System

Windows

### Organization

UDEx

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by PetyaMarkovaBogdanova** [comment]
Hi @aleksandra-beczala,
can you be more specific, exactly which ui5-carousel example did you test, with a link t the playground if I can ask.
Thank you!
Regards,
Petya Markova.
UI5 Web Components Dispatcher

**Comment by aleksandra-beczala** [comment]
Hi @PetyaMarkovaBogdanova,

I tried with simple example:
https://ui5.github.io/webcomponents/play/#eNqlU11r2zAU_Suqn9oQWU3LYHjOGM22p8EGax8GflGkG1upPowlJw0h_31XilOcdt1LscHWufcenfu1z5SV8JQ3weis2GeWG8iKMTjNhLMBbEC4vKCUtJrv6s71VtKV05JQ-rmy5cXXn4v7P7--kRgUgfglmtt6XmVgqwyx-JQNcIn_BN_Sh52GeJgU1oXLQsJKWZBXZI-YVD5eVRDrLHxC4BDjE3-yr1wnQFLhtOsol-vehzNfUrITfzwYCJyIhnceAkp6uP9OPyZVgylmjvhGwbZ1XagyMuSN4FbJ0MwlbJQAmg5ToqwKimvqBdcwn-XXA1lQAe_8zU2roWTHU8S1so-kA410SZZvAOItTQcrxHJmuLK58D7xlGyoE_4undyRFIN-Sy4eh-qnzAuy4d0lpZ63d8-mRbRcDYL-0TMKduhbdOjVByo4Wj3oFKFMTXwn8LYmhNYXjKFLXqvQ9MtcObaFpXCmxVLb4JkyvAbPfMp4lq_bGrPiOtbtB7cS69MCmQ1i3kF98wb1zfupb8-o73q9HDjZq9q8uQE4zaJTbSBh18ZGGSd7DUh61JS6u47NxbFMjqnLsbfHLrPT4vy3X9lheqIaresJGe8qm4xZGiWBTFhllYnTTeLAjcy-byOcBOL6vI5NCs7iv2BpXtQUNzawxVCtI9ezXhzsl4IjNFYcGxgX2_An2oCqG9zn2-sODCo6ZIfDX6f8neY

` <ui5-carousel navigation-placement="bottom">
 <img src="https://ui5.github.io/webcomponents/images/sample1.jpg" alt="Landscape 1">
 <img src="https://ui5.github.io/webcomponents/images/sample2.jpg" alt="Landscape 2">
 <img src="https://ui5.github.io/webcomponents/images/sample3.jpg" alt="Bulb">
 </ui5-carousel>`

**Comment by PetyaMarkovaBogdanova** [comment]
Hi colleagues from @UI5/ui5-team-ctr-rod ,
I reproduced the issue with https://stackblitz.com/edit/js-1rtxbh running Axe tool full page analysis.
The issue is related to that finding "Ensure an element's role supports its ARIA attributes" and the aria-activedescendant attribute on the root node and its "list" role.

Best regardws,
Petya Markova.
UI5 Web Componens Dispatcher.

**Comment by GerganaKremenska** [comment]
Hello @aleksandra-beczala, 

ui5-carousel had an ux redisgn in reagrds to keyboard handling and accessibility, the issues were resolved with this pull request  - https://github.com/UI5/webcomponents/pull/12256
